db.js

```diff
@@ -28,8 +31,38 @@ if (couchUrl) {
     }
 
     module.exports.fti = function(index, data, cb) {
-        var uri = path.join('/_fti/local', module.exports.settings.db, '_design', module.exports.settings.ddoc, index);
-        module.exports.request({ path: uri, qs: data }, cb);
```

Comment:
Did `module.exports.request` ever have a value? I'm so confused by this old code!

Reply:
This line is confusing and easy to miss: `module.exports = nano(baseUrl);`. This means that `modules.exports.request` is set to the `request` function in nano. We then extend nano with our own properties like `fti` and `settings`. It's a terrible pattern IMO because it's really hard to read and you risk overwriting something important in nano.

Reply:
😱